src/app/follow-button/follow-button.component.ts

```diff
@@ -222,6 +105,13 @@ export class FollowButtonComponent implements OnInit, OnDestroy {
   }
 
   ngOnInit() {
-    this.isFollowing = this._isLoggedInUserFollowing();
+    this.followService = new FollowService(
```

Comment:
No need to construct this service. Just add it as an argument in the constructor and angular will handle it. Instead, the methods in followService need to take `this.followedPubKeyBase58Check` as a param